files/fr/web/css/_colon_future/index.html

```diff
@@ -0,0 +1,66 @@
+---
+title: ':future'
+slug: 'Web/CSS/:future'
+browser-compat: css.selectors.future
+translation_of: Web/CSS/:future
+---
+<p>{{CSSRef}}</p>
+
+<p>Le sélecteur de <a href="/fr/docs/Web/CSS/Pseudo-classes">pseudo-classe</a> <a href="/fr/docs/Web/CSS">CSS</a> <strong><code>:future</code></strong> est une pseudo-classe agissant dans la dimension temporelle qui cible n'importe quel élément apparaîssant entièrement après un élément correspondant à {{cssxref(":current")}}. Ce sélecteur peut par exemple servir dans le cas d'une vidéo ayant des sous-titres affichés à l'aide du format <a href="/fr/docs/Web/API/WebVTT_API">WebVTT</a>.</p>
```

Comment:
Minor typo for "apparaîssant" > "apparaissant"

Reply:
Good point! Fixed in https://github.com/mdn/translated-content/pull/1718/commits/a8e0c31c8ac99facb822dd583ecfcfdd77682367